README.md

```diff
@@ -326,15 +326,30 @@ const weakMap = new WeakMap();
 weakMap.set(constObj, true);
 ```
 
-## FAQ
+# FAQ
 
-### Const classes?
+## Why #{}/#[] syntax? What about an existing or new keyword?
 
-Const classes are being considered as a followup proposal that would let us associate methods to const objects.
+Using a keyword as a prefix to the standard object/array literal syntax presents issues around
+backwards compatibility. Additionally, re-using existing keywords can introduce ambiguity.
 
-You can see an attempt at defining them in an [earlier version of this proposal](./history/with-classes.md).
+ECMAScript defines a set of *reserved keywords* that can be used for future extensions to the language.
+Defining a new keyword that is not already reserved is possible, but requires significant effort to validate
+that the new keyword will not likely break backwards compatibility.
+
+Using a reserved keyword makes this process easier, but is not a perfect solution, because there are no reserved keywords
+that match the "intent" of the feature, other than `const`. The `const` keyword is also tricky, because it describes
+a similar concept (variable reference immutability) while this proposal intends to add new immutable data structures.
+While immutability is the common thread between these two features, the features are orthogonal, and using `const`
+in both contexts would create hard-to-reason-about ambiguities.
 
-### Relation to const?
+Using the `#` character solves most of these problems. The `#` character can be safely used in this context without
+re-using an existing keyword or reserving a new keyword. Further, while `#` is already used to describe
+private fields in the [Class field declarations](https://github.com/tc39/proposal-class-fields) proposal, it is
+less ambiguous than re-using `const`, because they are completely orthogonal features (and contexts) with no relation
```

Comment:
This is a very subjective claim; i find it more ambiguous *because* they’re not related features.

Reply:
I agree, maybe saying that `const` had a strong community pushback would be more accurate

Reply:
I agree, probably a little too much personal opinion, not enough fact. I've reworded it to better fit @rricard's suggestion of referencing the community pushback against `const` alone.